setup.py

```diff
@@ -66,7 +66,7 @@
         "eth-keys>=0.4.0",
         "eth-rlp>=0.3.0",
         "eth-utils>=2.0.0",
-        "hexbytes>=0.1.0",
+        "hexbytes>=0.1.0,<=0.3.1",
```

Comment:
If we plan on making the breaking change in `0.4.0`, we should use `<0.4.0` here. That way if we want to backport any changes to `0.3.x`, we can. Thoughts?